php8.5-build/Dockerfile

```diff
@@ -0,0 +1,42 @@
+#ddev-generated
+ARG BASE_IMAGE
+FROM $BASE_IMAGE
+
+# Install system dependencies and PHP extensions that are commonly needed
+RUN apt-get update && apt-get install -y \
+    git \
+    unzip \
+    libzip-dev \
+    libpng-dev \
+    libjpeg-dev \
+    libfreetype6-dev \
+    libmagickwand-dev \
+    libicu-dev \
+    libxml2-dev \
+    libonig-dev \
+    && docker-php-ext-configure gd --with-freetype --with-jpeg \
+    && docker-php-ext-install -j$(nproc) \
+        bcmath \
+        curl \
+        dom \
```

Comment:
`dom` and `opcache` seem to have failures. Comment them out and then see if you can figure out why they don't build on debian bookworm. Alternately switch to debian trixie and see if those will build there.